docs/activities/tf-td/index.html

```diff
@@ -4,8 +4,59 @@
 group: "navigation"
 ---
 <div>
-	<h2>W3C WoT – ThingDescription Task Force</h2>
-	<p>
-		Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
-	</p>
-</div>
+	<h2>W3C WoT – Thing Description Task Force</h2>
+		<p>The WoT Thing Description task force is responsible for 
+		defining the information model for the Things, its interpretation and
+		common representation. In addition, the task force covers WoT Binding Templates 
+		topics to define the mapping to concrete IoT protocols and payload encodings.		
+		</p>
+		<div>
+		<h3>Normative Deliverables</h3>
+			<p>Current work is focused on a normative deliverables (WoT Thing Description 1.1) and on a informative deliverables (WoT Binding Templates).
+			</p>
+			<div>
+			<h4>WoT Thing Description 1.1</h4>
+				<p>This specification describes a superset of the features defined in Thing Description 1.0.
+					In general, the specification describes the formal model of a Thing and its common representation in JSON-LD 1.1. 
+					It introduces a simple interaction model with Properties, Actions, and Events
+				   to describe the capabilities of the Thing, including its data model, the communication protocol used, security mode, 
+				   and other semantic metadata. The WoT Thing Description 1.1 also formally introduces the Thing Model concept that has less 
+				   restriction and does not contain any instance-specific information.
+				</p>
+			</div>
+			<div>
+				<h4>WoT Binding Templates</h4>
+				<p>This document describes the set of vocabulary extensions to 
+				the WoT Thing Description that make up the Binding Templates. Binding 
+				Templates enable a Thing Description to be adapted to the specific protocol 
+				or data payload usage across the different standards. This is done through additional 
+				descriptive vocabulary that is used in the Thing Description. 
+				</p>
+        	</div>
+		</div>
+		<div>
+			<h3>People</h3>
+		<dl>
+			<dt>TF-Lead:</dt><dd>
+				Sebastian Kaebisch (Siemens AG)
+			</dd>
+			<dt>WoT Thing Description Co-Editors:</dt><dd>
+				Sebastian Kaebisch (Siemens AG),
+				Takuki Kamiya (Fujitsu Laboratories of America),
+				Victor Charpenay (Siemens AG)
+			</dd>
+			<dt>WoT Binding Templates:</dt><dd>
+				Michael Koster (SmartThings),
+				Ege Korkan (Siemens AG) 
+			</dd>
+		</dl>
+		</div>
+		<div>
+			<h3>Resources</h3>
```

Comment:
Shall we add links to the following 2 documents also?
* https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/
* https://www.w3.org/TR/2020/NOTE-wot-binding-templates-20200130/

Reply:
Discovery and Architecture have not this information.